matt-gabe/scripts/article.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var articles = [];
+
+function Article (rawDataObj) {
+  this.author = rawDataObj.author;
+  this.authorUrl = rawDataObj.authorUrl;
+  this.title = rawDataObj.title;
+  this.category = rawDataObj.category;
+  this.body = rawDataObj.body;
+  this.publishedOn = rawDataObj.publishedOn;
+}
+
+Article.prototype.toHtml = function() {
+  // DONE: Use handlebars to render your articles.
+  //       - Get your template from the DOM.
+  //       - Now "compile" your template with Handlebars.
+  var sourceHtml = $('#article-template').html();
+  var actualTemplate = Handlebars.compile(sourceHtml);
+
+  // REVIEW: If your template will use properties that aren't on the object yet, add them.
+  //   Since your template can't hold any JS logic, we need to execute the logic here.
+  //   The result is added to the object as a new property, which can then be referenced by key in the template.
+  //   For example, you might want to display how old a post is, or say "(draft)" if it has no publication date:
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  this.publishStatus = this.publishedOn ? `published ${this.daysAgo} days ago` : '(draft)'; // this is a "ternary operator"
+
+  // DONE: Use the function that Handlebars gave you to return your filled-in html template for THIS article.
```

Comment:
this is unnecessary, you just need the return statement `return actualTemplate`

Reply:
That was from the starter code. We weren't sure what to do with it.

Reply:
sorry for the confusion, I was talking about lines 29-37.